src/fidesops/ops/models/privacy_request.py

```diff
@@ -730,6 +731,17 @@ def hash_value(
         return hashed_value
 
 
+class Consent(Base):
+    """The DB ORM model for Consent."""
+
+    provided_identity_id = Column(String, ForeignKey(ProvidedIdentity.id), unique=True)
+    data_use = Column(String)
+    data_use_description = Column(String)
+    opt_in = Column(Boolean, nullable=False)
+
+    provided_identity = relationship(ProvidedIdentity, back_populates="consent")
```

Comment:
Since the unique constraint is defined on this table, I might define both sides of the relationship here with a backref, instead of using `back_populates` on both models, something like: 
```python3
provided_identity = relationship("ProvidedIdentity", backref=backref("consent", uselist=False))
```

and then the `consent` definition could be removed from ProvidedIdentity.   This is just a matter of preference though, in tending to define relationships on one side.

Reply:
Good testing additions might be model-level tests that assert that both sides of the relationship are defined properly, asserting both `provided_identity.consent` and `consent.provided_identity`. We have a tests > models folder.

Reply:
I originally had `backref` and changed it because I saw [this](https://docs.sqlalchemy.org/en/14/orm/backref.html) in the sqlalchemy docs where is say `backref` is legacy now and `back_populates` should be preferred. Thoughts?

Reply:
ah thanks I wasn't current with this latest recommendation.